SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -416,9 +428,10 @@ def customise_Reco(process,pileup):
     process.pixelTracks.FilterPSet.chi2 = cms.double(50.0)
     process.pixelTracks.FilterPSet.tipMax = cms.double(0.05)
     process.pixelTracks.RegionFactoryPSet.RegionPSet.originRadius =  cms.double(0.02)
-    process.templates.DoLorentz=False
-    process.templates.LoadTemplatesFromDB = cms.bool(False)
-    process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)
+    # use defaults d.k. 2/16
+    #process.templates.DoLorentz=False
+    #process.templates.LoadTemplatesFromDB = cms.bool(False)
+    #process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)

```

Comment:
I can really only speak for (phase 1) tracking, and we won't use the the customization alone without the era.

As I wrote earlier, I don't know what other parties are involved. I would guess the customize to be of interest only for pixel offline and tracking at the moment, but who knows... My personal opinion is that the customize will go away "soon", and if something is already migrated to eras it is probably not worth to update the corresponding part of the customize.